engine/src/main/java/org/terasology/rendering/opengl/GraphicState.java

```diff
@@ -318,7 +345,7 @@ public void preRenderSetupLightGeometry() {
         glDisable(GL_DEPTH_TEST);
 
         glEnable(GL_BLEND);
-        glBlendFunc(GL_ONE, GL_ONE);
+        glBlendFunc(GL_ONE, GL_ONE_MINUS_SRC_COLOR);
```

Comment:
Tricky, tricky, tricky. Again here there's a problem of simulated physics and perception. If I place 20 torches in a place where there was only one, the number of photons is multiplied by 20 and the increase in brightness is justified.  What is not justified is 
1. the sharply burned areas: if they get to be so white bright bloom should kick in and, at least initially, make everything or at least some areas one big bright blob. Eventually however the eye should adapt and everything should look pretty normal, in the general direction of the second image, OR we could decide that depending on the amount of light it stays bright and blinding: after all, if we look into the sun we -DO NOT- adapt to its brightness.
2. the other problem is the pitch black dark surfaces generating a strong sense of contrast with the burned surfaces. That of course doesn't happen in reality because photons bounce around and eventually would reach most surfaces in this kind of setup. This problem I feel it's harder to solve. Can thrown torches be plugged in the light propagation system? Or could they perhaps dimly light up also surfaces that are pointing away from them?

Again, all this is probably out of the scope of this particular PR but I mention it because we might one day have to undo these changes when addressing the whole simulated lighting/simulated perception situation as a whole.